http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=539193

From: Santiago Vila <sanvila@unex.es>

The current code uses /usr/lib/sendmail and then /usr/sbin/sendmail as
a fallback. Policy says it should use /usr/sbin/sendmail.

Index: libmailtools-perl-2.13/lib/Mail/Mailer.pm
===================================================================
--- libmailtools-perl-2.13.orig/lib/Mail/Mailer.pm
+++ libmailtools-perl-2.13/lib/Mail/Mailer.pm
@@ -22,7 +22,7 @@ sub is_exe($);
 sub Version { our $VERSION }
 
 our @Mailers =
-  ( sendmail => '/usr/lib/sendmail;/usr/sbin/sendmail;/usr/ucblib/sendmail'
+  ( sendmail => '/usr/sbin/sendmail;/usr/lib/sendmail;/usr/ucblib/sendmail'
   , smtp     => undef
   , smtps    => undef
   , qmail    => '/usr/sbin/qmail-inject;/var/qmail/bin/qmail-inject'
